### ISSUE TITLE
No coverage reported with CoffeeScript 2 compiler and Mocha

### ISSUE THREAD

**Issue by bergie** [benbria/coffee-coverage#90]
If I run my tests with CoffeeScript 2 compiler, I get zero coverage reported:

```bash
$ nyc mocha --require coffeescript/register --require coffee-coverage/register-istanbul spec/*.coffee
```

With CoffeeScript 1.x compiler I get the expected reports:

```bash
$ nyc mocha --require coffee-script/register --require coffee-coverage/register-istanbul spec/*.coffee
```

**Comment by jwalton** [comment]
I haven't looked into CoffeeScript 2.x at all, I'm afraid.

**Comment by etamponi** [comment]
You might try https://github.com/benbria/coffee-coverage/pull/91. I've not tested it extensively, but the tests pass and it runs on my project, so perhaps it's enough for you too?

**Comment by baoshan** [comment]
Hello everyone, any plan on 2.x support? Can #91 be accepted?

**Comment by jwalton** [comment]
Sorry, I've been crazy busy and haven't had a chance to even look at #90.
If I don't get it merged by end of day Tuesday, harass me some more
please.  :)

On Dec 8, 2017 22:27, "Baoshan Sheng" <notifications@github.com> wrote:

Hello everyone, any plan on 2.x support? Can #91
<https://github.com/benbria/coffee-coverage/pull/91> be accepted?

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/benbria/coffee-coverage/issues/90#issuecomment-350420514>,
or mute the thread

**Comment by baoshan** [comment]
Be rest assured I’ll harass you relentlessly. :)

**Comment by igoramadas** [comment]
+1 !!!

**Comment by jwalton** [comment]
Fixed via https://github.com/benbria/coffee-coverage/pull/91, published as v3.0.0.
